src/Tasks/Hash.cs

```diff
@@ -31,6 +33,14 @@ public class Hash : TaskExtension
         /// </summary>
         public bool IgnoreCase { get; set; }
 
+        /// <summary>
+        /// When true, will generate a hash that does not depend on the order of the items.
+        /// </summary>
+        /// <remarks>
+        /// When true, the items are sorted first using their Identity
+        /// </remarks>
+        public bool IgnoreOrder { get; set; }
```

Comment:
IgnoreOrder is somewhat confusing here, seeing as it does an explicit sort. A name that suggests "a sorted hash" would be more intuitive.

Reply:
It is much similar to IgnoreCase. In order to ignore the case the code does explicit ToUpper. That is how we ignore it - we normalize it.
The same is with IgnoreOrder. 
To the users of the task it will be natural and obvious, they do not care about the implementation details.

Reply:
I'm convinced!